app/models/backend_event.rb

```diff
@@ -1,7 +1,5 @@
 class BackendEvent < ApplicationRecord
 
-  self.primary_key = :id
```

Comment:
This is quite confusing :upside_down_face: 

I am wondering what was the intention here, and why the default `id` (as created by Rails) was not used here.

Reply:
I think they wanted a primary key, but not an autonumeric `id`, which is the default in Rails. So they did a hack to have something equivalent to a PK without the autonumeric. It seems to work that way now, probably because of the `default: nil` added to `create_table` [here](https://github.com/3scale/porta/pull/3476/files#diff-cbf8b25d6d853acf58fa9057841f407d29ffe90649c75cf9e3f51b2d6e3aa1d3R235)